LondonBritishLibrary/orient/BLorient566.xml

```diff
@@ -31,74 +31,147 @@ schematypens="http://relaxng.org/ns/structure/1.0"
-                                
+                                <foliation>Blank leaves are unnumbered</foliation>
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns="2" writtenLines="19 20"/>
                             </layoutDesc>
                         </objectDesc>
-                        
                         <handDesc>
-                            
                             <handNote script="Ethiopic" xml:id="h1">
-                           
                                 <desc>The manuscript is written in a good hand of the,  <date>19th century</date>.
                                 </desc>
-                                
-                                <date notBefore="1800" notAfter="1900"/>
-                                
+                                <date notBefore="1800" notAfter="1812"/>
                             </handNote>
-                            
-                        </handDesc>   
+                        </handDesc>
+                        <decoDesc>
+                          <decoNote xml:id="d1" type="ornamentation">
+                            <locus target="#1r"></locus>
+                            <desc>Polychrome ornamental band</desc>
+                          </decoNote>
+                          <decoNote xml:id="d2" type="ornamentation">
+                            <locus target="#5va"></locus>
+                            <desc>Small polychrome ornamental band</desc>
+                          </decoNote>
+                          <decoNote xml:id="d3" type="ornamentation">
+                            <locus target="#41r"></locus>
+                            <desc>Polychrome ornamental band</desc>
+                          </decoNote>
+                          <decoNote xml:id="d4" type="ornamentation">
+                            <locus target="#55vb"></locus>
+                            <desc>Small polychrome ornamental band</desc>
+                          </decoNote>
+                          <decoNote xml:id="d5" type="ornamentation">
+                            <locus target="#57rb"></locus>
+                            <desc>Small polychrome ornamental band</desc>
+                          </decoNote>
+                          <decoNote xml:id="d6" type="ornamentation">
+                            <locus target="#71ra"></locus>
+                            <desc>Small polychrome ornamental band</desc>
+                          </decoNote>
+                        </decoDesc>
                         <additions>
-                            
+                          <list>
+                            <item xml:id="e1">
+                              <locus target="#1 #6 #11 #18 #30 #36 #38 #39 #40 #41 #55 #57 #71"></locus>
+                              <desc type="findingAid">Finding aids, colored pieces of threads in blue
```

Comment:
I think that these are leaf string markers and should be encoded in the description of the binding